### PR TITLE
add descripiton and keywords to package.json

### PR DESCRIPTION
**Pull request by juliannzhou**
This PR adds description and keywords to package.json. The description is copied from README and keywords provided by product.

J=SLAP-2190
TEST=manual

### CODE REVIEW

**Review by oshi97 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
   "main": "./dist/commonjs/src/index.js",
   "module": "./dist/esm/src/index.js",
+  "keywords": [
+    "search",
+    "nlp",
+    "ai",
+    "answers",
+    "yext",
+    "react",
+    "components"
```
Comment (oshi97) [inline package.json:16]: Did product ask for "components" and "react" here? This library doesn't have any components in it, and also doesn't use react

Comment (oshi97) [reply package.json:16]: I would probably put "redux" here if anything

**Review by nmanu1 — COMMENTED** [review]
can this PR be merged into `develop` instead of `main`

File: package.json (hunk 1 above)
Comment (juliannzhou) [reply package.json:16]: Product only gave me one list, but none of the repositories have keywords in package.json. I thought I'd just put the same list, but I was wondering if we need different lists of keywords for all of the repositories?

Comment (juliannzhou) [reply package.json:16]: I merged this branch by mistake, but I reverted the changes...

Comment (juliannzhou) [reply package.json:16]: I merged this into main by mistake, but I've reverted the changes.

**Pull request merged**